rest/taskana-rest-spring/src/main/java/pro/taskana/rest/common/TaskanaErrorData.java

```diff
@@ -25,7 +25,7 @@
     this.exception = ex.getClass().getName();
     this.message = ex.getMessage();
     this.path = req.getDescription(false);
-    if (this.path != null && this.path.startsWith("uri=")) {
+    if (this.path.startsWith("uri=")) {
```

Comment:
why did you remove the null check?

Reply:
According to the IntelliJ and Sonarcloud the method getDescription never returns null and != null is thereby always true.